CalSmokeApp/Gemfile

```diff
@@ -8,7 +8,7 @@ gem "json", "1.8.6"
 # workflow and build tools
 gem 'retriable', '~> 2.0'
 gem 'rake', '~> 10.3'
-gem 'bundler', '~> 1.6'
+gem 'bundler', '~> 2.0'
```

Comment:
bundler 1.17.3

Reply:
This bundler version make a mistake in ADO: https://msmobilecenter.visualstudio.com/Mobile-Center/_build/results?buildId=782779

Reply:
Done!